lib/wovnrb.rb

```diff
@@ -131,6 +132,12 @@ def output(headers, status, res_headers, body)
       [status, res_headers, body]
     end
 
+    def should_translate?(dom, values)
+      translatable_html = dom.html? ? true : @store.settings['translate_fragment']
```

Comment:
I'd prefer `could_translate` instead of `translatable_html` because of `@store.settings['translate_fragment']` usage here

Reply:
Agree, I'll change, thanks